docs/reasons.md

```diff
@@ -17,6 +17,10 @@ maintain system performance.
   - *Summary*: failed to broadcast duty to beacon node
   - *Details*: Reason `bcast` indicates that beacon node returned an error while submitting aggregated duty signature to beacon node.
 
+### Failure Reason: `chain_inclusion`
+  - *Summary*: duty not included on-chain
+  - *Details*: Reason `chain_inclusion` indicates that even though charon broadcasted the duty successfully, it wasn`t included in the beacon chain. This is expected for up to 20% of attestations. It may however indicate problematic charon broadcast delays or beacon node network problems.
```

Comment:
nit: do we have data that only 80% attestations end up on-chain?

Reply:
Although that might not be accurate, here is better data. Seems like 5% is more realistic for goerli at least. Well see how mainnet does after releasing v0.16.

<img width="1502" alt="image" src="https://github.com/ObolNetwork/charon/assets/29249923/2ad2e529-daf0-4949-9588-5f2af2aec43a">